app/src/main/java/com/example/lhi06/residue/FindAlbumActivity.java

```diff
@@ -58,6 +59,9 @@ public void onFailure(final Call<AlbumSearchResponse> call, final Throwable t) {
 
         } catch (IOException e) {
             e.printStackTrace();
+        } catch (NullPointerException e) {
```

Comment:
Don't think you will get a NullPointerException if the user doesn't enter any text